### DISCUSSION TITLE
popper element going behind overflow: hidden container

### DISCUSSION THREAD

**Discussion by rokin04** [floating-ui/floating-ui#1210] [Q&A]
Hi Team

I'm using popper inside react-table and each cell has overflow: hidden property and I want to render popper element inside those cell I'm using strategy: fixed but that's not helping to show popper on top of the container 
anybody know how to handle this ?

**Comment by FezVrasta** [comment]
You should render the popper element outside the cells

**Reply by rokin04** [reply]
Isn't there anyway to achieve this by rendering popper inside cell?

**Reply by FezVrasta** [reply]
You should make so that the cell is not a clipping container. It will need its CSS to be changed, I'm not sure if it's possible in your case.

**Reply by rokin04** [reply]
I just made some changes to popper element added strategy: fixed and rendering it inside cells, now part of popper is visible in the reference cell but other part of it going behind below cells.

**Reply by rokin04** [reply]
I got the solution :)

**Reply by celine3d** [reply]
Hi @rokin04  could you please share the solution? I have similar problem and nothing seems to work for me...

**Reply by rokin04** [reply]
I think your popper z-index is less than the cell.

**Reply by Fausto-Payano-Axomic** [reply]
@celine3d If changing the Z-index doesn't help you try using a portal element located outside of the container with the style property `overflow: hidden`.

**Reply by karelkangro** [reply]
-- moved my message to the new thread

**Reply by simplenotezy** [reply]
Would love to know the solution as well.

**Reply by devgioele** [reply]
For anyone coming from a search engine, I fixed it using a `FloatingPortal`:
```tsx
export function Menu({ className, children, button, placement }: MenuProps) {
  const { menu } = useTheme()

  const {
    refs,
    floatingStyles,
    placement: finalPlacement,
  } = useFloating({
    whileElementsMounted: autoUpdate,
    placement,
    strategy: 'absolute',
    middleware: [offset(remToPx(1)), flip({ padding: remToPx(1) })],
  })

  return (
    <HeadlessMenu as="div" className={classNames(menu.container, className)}>
      {({ open }) => (
        <>
          <HeadlessMenu.Button as={Fragment} ref={refs.setReference}>
            {typeof button === 'function'
              ? button({
                  placement: placementUnionToPlacementEnum(finalPlacement),
                  open,
                })
              : button}
          </HeadlessMenu.Button>
          {open && (
            <FloatingPortal>
              <HeadlessMenu.Items
                ref={refs.setFloating}
                className={menu.list.base}
                style={floatingStyles}
              >
                {children}
              </HeadlessMenu.Items>
            </FloatingPortal>
          )}
        </>
      )}
    </HeadlessMenu>
  )
}
```
